js/app.js

```diff
@@ -78,49 +76,58 @@ angular.module('PathOfDamage', [])
     if (hit === null) {
       return {};
     }
-    var physDamage = hit;
+
+    var damage = {
```

Comment:
The calcDamage method is now a 60 line monstrosity hanging out in the middle of our main app controller. I don't exactly know why that bothers me, but I think I would like to eventually make it a service with its own file, and/or break out the subsections of the calculations into methods. I'd like to have good comments explaining what we are doing, but right now that would make this method even larger. I think it's fine for this review, but we should keep it in mind.